common/buildings/00_tradepost.txt

```diff
@@ -463,3 +463,294 @@ trade_post = {
 		# extra_tech_building_start = 10.0 # Never get at start of game
 	# }
 }
+
+##########################################
+#
+# Ciphers Trade Posts mod merger buildings
+#
+##########################################
+
+trade_post = {	
+	
+#Garrison
+
+	#StableMaster
+	wc_stableMaster = {
+		 desc = wc_stableMaster1_desc
+		 potential = {
+			always = yes
+		 }
+		 gold_cost = 650
+		 build_time = 365
+		 local_tax_modifier = 0.1
+		 trade_route_wealth = 0.5
+		 ai_creation_factor = 100
+	}
+	
+	#FlightMaster
+	wc_flightMaster = {
```

Comment:
This building should not be in non-trade route posts. `trade_route_wealth` isn't used in republic trade posts. `local_tax_modifier` may give bonus to provinces you don't own.